docs/examples/track_fusion_example.py

```diff
@@ -0,0 +1,628 @@
+PF_track1, PF_track2, KF_track1, KF_track2 = set(), set(), set(), set()
+PF_fused_track, KF_fused_track = set(), set()
+
+# Instantiate the various trackers using the general_tracker function,
+# we assign a unique tracker the detections from a specific radar simulator
+KF_tracker_1 = general_tracker(SingleTargetTracker, r1simulators[0], KF_updater,
+                                 initiator, deleter, data_associator_KF)
+
+KF_tracker_2 = general_tracker(SingleTargetTracker, r2simulators[0], KF_updater,
+                               initiator, deleter, data_associator_KF)
+
+PF_tracker_1 = general_tracker(SingleTargetTracker, r1simulators[1], PF_updater,
+                             PF_initiator, deleter, data_associator_PF)
+
+PF_tracker_2 = general_tracker(SingleTargetTracker, r2simulators[1], PF_updater,
+                               PF_initiator, deleter, data_associator_PF)
+
+# Load the detection generator
+g1 = radar_simulator1.detections_gen()
+g2 = radar_simulator2.detections_gen()
+
+# Each tracker will have the same data
+gg1 = deepcopy(radar_simulator1).detections_gen()
+gg2 = deepcopy(radar_simulator2).detections_gen()
+
+
+# Loop on the simulation steps
+for aa in range(number_of_steps):
+
+    radar1_detections = next(g1)
+
+    # Run the Particle Filter tracker on the radars measurements
+    PF_tracker_1.detector = DummyDetector(current=radar1_detections)
+    PF_tracker_1.__iter__()
+    _, PF_sensor_track1 = next(PF_tracker_1)
+    PF_track1.update(PF_sensor_track1)
+
+    rd1 = next(gg1)
+    # Run the Kalman Filter tracker on the radars measurements
+    KF_tracker_1.detector = DummyDetector(current=rd1)
+    KF_tracker_1.__iter__()
+    _, KF_sensor_track1 = next(KF_tracker_1)
+    KF_track1.update(KF_sensor_track1)
+
+    radar2_detections = next(g2)
+
+    PF_tracker_2.detector = DummyDetector(current=radar2_detections)
+    PF_tracker_2.__iter__()
+    pf_time, PF_sensor_track2 = next(PF_tracker_2)
+    PF_track2.update(PF_sensor_track2)
+
+    rd2 = next(gg2)
+
+    KF_tracker_2.detector = DummyDetector(current=rd2)
+    KF_tracker_2.__iter__()
+    kf_time, KF_sensor_track2 = next(KF_tracker_2)
+    KF_track2.update(KF_sensor_track2)
+
+    # We have now the track for each radar now let's perform the
+    # track fusion
+    for PF_track_meas in [PF_track1, PF_track2]:
+        dummy_detector_PF = DummyDetector(current=[pf_time, PF_track_meas])
+        track_fusion_tracker2.detector = Tracks2GaussianDetectionFeeder(dummy_detector_PF)
+        track_fusion_tracker2.__iter__()
+        _, tracks = next(track_fusion_tracker2)
+        PF_fused_track.update(tracks)
+
+    for KF_track_meas in [KF_track1, KF_track2]:
+        dummy_detector_KF = DummyDetector(current=[kf_time, KF_track_meas])
+        track_fusion_tracker.detector = Tracks2GaussianDetectionFeeder(dummy_detector_KF)
+        track_fusion_tracker.__iter__()
+        _, tracks = next(track_fusion_tracker)
+        KF_fused_track.update(tracks)
```

Comment:
This line is causing the first cell at the start of Section "3) Run the trackers..." to fail on build with the following errors;
On CircleCI "ValueError: Mime type rendering requires ipython but it is not installed"
On local build "ValueError: All detections must have the same timestamp"